corehq/apps/locations/fixtures.py

```diff
@@ -87,6 +103,34 @@ def __call__(self, restore_user, version, last_sync=None, app=None):
         return [root_node]
 
 
+class FlatLocationSerializer(object):
+    id = 'locations'
+
+    def get_xml_nodes(self, restore_user, all_locations):
+        if not toggles.FLAT_LOCATION_FIXTURE.enabled(restore_user.project.name):
```

Comment:
not really that important since project is memoized, but more as an fyi the restore user has a `domain` property that is exactly what this is